pkg/kopia/repository/client.go

```diff
@@ -50,7 +50,10 @@ func ConnectToAPIServer(
 	// Extra fingerprint from the TLS Certificate secret
 	fingerprint, err := kopia.ExtractFingerprintFromCertificate(tlsCert)
 	if err != nil {
-		return errors.Wrap(err, "Failed to extract fingerprint from Kopia API Server Certificate Secret")
+		fingerprint, err = kopia.ExtractFingerprintFromCertificateJSON(tlsCert)
```

Comment:
Is there a way to check the cert format and have switch case accordingly to parse the cert as per the type?

Reply:
Yes will implement that

Reply:
we can't use switch directly on tlsCert as in both case they are in string format